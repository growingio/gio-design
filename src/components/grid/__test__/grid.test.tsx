import React from 'react';
import { render } from '@testing-library/react';
import { Row, Col } from '..';

describe('Testing Grid', () => {
  it('should be stable', () => {
    const { asFragment } = render(
      <Row gutter={[10, 10]}>
        <Col span={3}>0</Col>
        <Col order={1}>3</Col>
        <Col>1</Col>
        <Col span={7}>
          <Row>
            <Col span={6} offset={3}>
              4
            </Col>
            <Col span={3}>5</Col>
          </Row>
        </Col>
      </Row>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
