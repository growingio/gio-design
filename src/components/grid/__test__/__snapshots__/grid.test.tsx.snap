// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Grid should be stable 1`] = `
<DocumentFragment>
  <div
    class="gio-row"
    style="margin: -5px -5px 5px -5px;"
  >
    <div
      class="gio-col gio-col-span-3"
      style="padding: 5px 5px 5px 5px;"
    >
      0
    </div>
    <div
      class="gio-col gio-col-span-1 gio-col-order-1"
      style="padding: 5px 5px 5px 5px;"
    >
      3
    </div>
    <div
      class="gio-col gio-col-span-1"
      style="padding: 5px 5px 5px 5px;"
    >
      1
    </div>
    <div
      class="gio-col gio-col-span-7"
      style="padding: 5px 5px 5px 5px;"
    >
      <div
        class="gio-row"
      >
        <div
          class="gio-col gio-col-span-6 gio-col-offset-3"
        >
          4
        </div>
        <div
          class="gio-col gio-col-span-3"
        >
          5
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
